webapp/src/features/adminAddNotification/AdminAddNotification.tsx

```diff
@@ -68,9 +63,18 @@ const AdminAddNotification = () => {
     dispatch(updateEmailTypesApiData())
   }, [apiData, dispatch])
 
+  const notifySuccess = (message: string) => toast.success(message)
+  const notifyError = (message: string) => toast.error(message)
+
   const onSubmit = (data: AdminNotificationForm) => {
     data.email = userEmail
-    dispatch(submitForm({ data, reset }))
+    dispatch(submitForm({ data, reset })).then((data: any) => {
+      data.payload.success
+        ? notifySuccess(data.payload.message)
+        : notifyError('Failed to add user notification due to error: ' + data.payload.message)
```

Comment:
I would like to keep the error messages displayed in-page (not toast notifications) for form-based validation errors